backend/alerts/integrations/discord/messages.go

```diff
@@ -47,6 +47,11 @@ func (bot *Bot) SendErrorAlert(channelId string, payload integrations.ErrorAlert
 	embed.Description = payload.UserIdentifier
 	embed.Fields = fields
 
+	sessionLabel := "View Session"
+	if payload.SessionExcluded {
+		sessionLabel = "No recorded session"
```

Comment:
i like it! the 'new tab' icon doesn't seem applicable here since it's disabled, but maybe we still keep it to be consistent with the rest of the buttons?